README.md

```diff
@@ -18,39 +18,30 @@ Originally developed by [Carlos Tadeu Panato Junior](https://github.com/cpanato/
 
```

Comment:
Propose adding a section of "Requirements" that include minimum server version (if applicable), license requirement etc.

Reply:
Minimum server version is `5.2.0`. 0/5 on if we need to mention this. This plugin does not require a paid license to run.

Reply:
I've taken a look through the doc again and I don't think we need to call out when something doesn't require a paid license. I'm 0/5 on mentioning the minimum server version. @hanzei, can you be the tie-breaker?

Reply:
I don't see a reason to call out requirements if there aren't any.